src/ubuntu/16.04/x86/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM i386/ubuntu:16.04
```

Comment:
What is the intended use of this?  This is quite different then the x64 Dockerfile.  I am wondering if this should be tagged more specific to the intended use.